src/test/static/class/class-methods.js

```diff
@@ -46,4 +46,14 @@ class Class {
   customInstanceFunctionWithParamsAndPrivateJSDoc() {
     return 8;
   }
+
+  /**
+   * This is the description for customInstanceFunctionWithRestParam.
+   * @param {Number} a The first argument.
+   * @param {...Number} b The second argument.
```

Comment:
I had to verify that `...number` is correct, rather than typescript's `number[]`.